.github/workflows/build.yml

```diff
@@ -6,30 +6,20 @@ jobs:
     timeout-minutes: 15
     runs-on: ubuntu-24.04
 
-    strategy:
-      matrix:
-        adapter: ['react', 'vue3', 'svelte']
-
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Install pnpm
+        uses: pnpm/action-setup@v3
+
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
           node-version: 22.14
```

Comment:
@pascalbaljet I think `cache: pnpm` was lost from here during merge.

Reply:
Fixed in c900397e